hikari/impl/bot.py

```diff
@@ -833,7 +829,7 @@ async def _set_close_flag(self, signame: str, signum: int) -> None:
         # without getting undefined behaviour. We do however have `asyncio.run_coroutine_threadsafe` which can
         # run a coroutine function on the event loop from a completely different thread, so this is the safest
         # solution.
-        _LOGGER.info("received interrupt %s (%s), will start shutting down shortly", signame, signum)
+        _LOGGER.debug("received interrupt %s (%s), will start shutting down shortly", signame, signum)
```

Comment:
This should also stay on info since it will provide feedback to windows users since `^C` doesn't get outputted to some terminals (cmd) when causing SIGINT

Reply:
it will output that the bot is shutting down anyway